app/src/main/java/com/google/droidjump/GameConstants.java

```diff
@@ -25,8 +25,14 @@
     int DROID_FIRST_STEP_INDEX = 5; // Index of droid at res/mipmap/droid.png image.
     int DROID_SECOND_STEP_INDEX = 6; // Index of droid at res/mipmap/droid.png image.
     int SLEEP_TIME = 15; // Sleep time in gameView (ms).
-    int GAME_LEVELS_COUNT = 100; // Count of game levels.
+    int GAME_LEVELS_COUNT = 4; // Count of game levels.
+    int GAME_DEFAULT_VALUE = 1; // Default value.
```

Comment:
Name of this constant does not really tell for what it is used. Maybe it should be "START_LEVEL_ID" or "FIRST_LEVEL_ID"?

Reply:
Fixed.